models/segment.schema.js

```diff
@@ -0,0 +1,20 @@
+const mongoose = require('mongoose');
+
+const segmentSchema = new mongoose.Schema({
+  _id: String,
+  name: {type: String, required: true, unique: true},
```

Comment:
🚫 **[eslint]** <[prettier/prettier](https://github.com/prettier/eslint-plugin-prettier#options)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Replace `type:·String,·required:·true,·unique:·true` with `·type:·String,·required:·true,·unique:·true·`
```suggestion
  name: { type: String, required: true, unique: true },
```

Reply:
My eslint on vscode tells me having a space here is an error and the husky hooks didn't give me an error here either, but the eslint here flags this as an error